internal/handler/handler.go

```diff
@@ -29,11 +30,22 @@ const (
 )
 
 type Input struct {
-	Groups     []string   `json:"groups"`
-	Permission Permission `json:"permission"`
-	Resource   string     `json:"resource"`
-	Subject    string     `json:"subject"`
-	Tenant     string     `json:"tenant"`
+	Groups     []string             `json:"groups"`
+	Permission Permission           `json:"permission"`
+	Resource   string               `json:"resource"`
+	Subject    string               `json:"subject"`
+	Tenant     string               `json:"tenant"`
+	Extras     InputExtraAttributes `json:"extras,omitempty"`
+}
+
+type InputExtraAttributes struct {
+	Logs InputLogsExtraAttributes `json:"logs,omitempty"`
+}
```

Comment:
Do we need this extra level of indirection? I suggest to remove it and omit mentioning the signal name entirely since this authorizer is used by other signals too. They might also want to extra selectors from their metadata fields and switch to namespaced SARs. WDYT?